### PR TITLE
Add pytorch migration now that GPU builds have been uploaded

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
@jakirkham @isuruf if you have time to merge this this would be greatly appreciated.
We are almost done building pytorch 1.9.1
https://github.com/conda-forge/pytorch-cpu-feedstock/pull/68

**Comment by hmaarrfk** [comment]
@isuruf should i bump this to 1.9.1?

**Comment by hmaarrfk** [comment]
@conda-forge/pytorch @conda-forge/core is anybody willing to review this?

A few packages are trying to find ad-hoc solutions for this since this pinning isn't available.

**Comment by h-vetinari** [comment]
> A few packages are trying to find ad-hoc solutions for this since this pinning isn't available.

Yup, also affected. Would be good to have a global pinning. :)

**Comment by h-vetinari** [comment]
@conda-forge/core 
Anything still blocking this? There are now 30+ packages that depend on pytorch (though not all compile against it), so it would be nice to not have to pin the pytorch-version between host & run and then not to forget to migrate manually.

**Comment by hmaarrfk** [comment]
@conda-forge/conda-forge-pinning-feedstock kind ping!

File: recipe/migrations/pytorch.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+migrator_ts: 1631819618
+__migrator:
+  kind: version
+  migration_number: 1
+  bump_number: 1
+
+pytorch:
+  - 1.10.0
```
Comment (isuruf) [inline recipe/migrations/pytorch.yaml:8]:
```suggestion
  - 1.10
```

**Review by isuruf — COMMENTED** [review]
Rename this file to `pytorch110.yaml`

**Comment by hmaarrfk** [comment]
thank you for the feedback

**Pull request merged**